docs/pages/OneFilter/OF-Levels.md

```diff
@@ -0,0 +1,127 @@
+
+![OneFilter select model view](../../../assets\images\OneFilter\OF-Lv-WholeModel.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+2. Now, use the checkboxes to select the categories to filter. Note that, once you have selected the categories, on the right table will show the existing families, with the category, family name, type name, Level and count columns.
+
+![OneFilter choose categories](../../../assets\images\OneFilter\OF-Lv-Categories.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+## Find Elements
+
+Create conditions rules to find elements between Levels and Reference Planes. Follow the steps:
+
+1. Select the AND/OR condition from the drop-down list
+
+2. Click Add Rule
+
+3. Select from the Reference level or Reference plane drop-down list
+
+Once you have selected it, define whether you want to find elements above or below the selected level/reference plane and then enter the name of the level/reference plane in the text input. Please note that in some cases the reference plane may not be renamed, so use the Select button to choose the reference plane manually.
+
+4. Repeat the last step by selecting the corresponding reference level/plane to find elements between the two selected situations.
+
+![OneFilter level/reference plane](../../../assets\images\OneFilter\OF-Lv-Rule.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+5. Click on the Filter button to execute the rule condition.
+
+![OneFilter apply filter level/reference plane](../../../assets\images\OneFilter\OF-Lv-Filter.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+6. Click the Clear button to reset the condition of the rule you created.
+
+![OneFilter clear level/reference plane](../../../assets\images\OneFilter\OF-Lv-Clear.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+### Search
+
+The search box will search for elements contained in the filtered elements table.
+
+![OneFilter search](../../../assets\images\OneFilter\OF-Lv-Search.gif)  
+<sub>Note: the version on the image may not reflect the [latest version of OneFilter/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
+
+---
+
+### Update or duplicate view
+
+Mark with checkbox elements that you want isolate with the Section Box. You have the ability to use existing or to duplicate 3D view. Visualize only filtered elements or all elements within Section Box with ability to define offset form marked elements.
```

Comment:
"Check the elements that you want isolate within a Section Box. You can use existing 3D Views or duplicate them. Visualize filtered elements or all elements within the Section Box, with the ability to define an offset from marked elements."